akka-docs-dev/rst/stages-overview.rst

```diff
@@ -0,0 +1,134 @@
+scan                   the function scanning the element finishes                                                                                  downstream backpressures                                                                                                        Upstream completes
+drop                   the specified number of elements has been dropped already                                                                   the specified number of elements has been dropped and downstream backpressures                                                  Upstream completes
+take                   the specified number of elements to take has not yet been reached                                                           downstream backpressures The specified number of elements has been emitted, or upstream completes                               The defined number of elements has been taken or upstream completes
+=====================  =========================================================================================================================   ==============================================================================================================================  =====================================================================================
+
+Asynchronous processing stages
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+These stages encapsulate an asynchronous computation, properly handling backpressure while taking care of the asynchronous
+operation at the same time (usually handling the completion of a Future).
+
+**It is currently not possible to build custom asynchronous processing stages**
+
+=====================  =========================================================================================================================   ==============================================================================================================================  =====================================================================================
+Stage                  Emits when                                                                                                                  Backpressures when                                                                                                              Completes when
+=====================  =========================================================================================================================   ==============================================================================================================================  =====================================================================================
+mapAsync               the Future returned by the provided function finishes for the next element in sequence                                      the distance between the first uncompleted Future and between the last processed Future is larger than the defined parallelism  Upstream completes [1]_
```

Comment:
Is it? Why would the unordered version wait for the last slot to become empty? It can inject the new Future in any of the free slots since it does not care about ordering.